packages/test-apps/nextjs/src/utils/getExperience.ts

```diff
@@ -0,0 +1,48 @@
+// fetchBySlug currently needs to be imported from core to avoid NextJS build errors
+import { fetchBySlug } from '@contentful/experiences-core';
+import { createClient } from 'contentful';
+
+const accessToken = process.env.NEXT_PUBLIC_CTFL_ACCESS_TOKEN!;
+const prevAccessToken = process.env.NEXT_PUBLIC_CTFL_PREVIEW_ACCESS_TOKEN!;
+const space = process.env.NEXT_PUBLIC_CTFL_SPACE!;
+const environment = process.env.NEXT_PUBLIC_CTFL_ENVIRONMENT!;
```

Comment:
inconsequential suggestion: could provide a default like 'master' similar to how you did for domain